docs/packages/omc3/workflow.md

```diff
@@ -1,7 +1,824 @@
+        observe, place="bpm.21r1.b1";
+        observe, place="bpm.22r1.b1";
+        observe, place="bpm.23r1.b1";
+        observe, place="bpm.24r1.b1";
+        observe, place="bpm.25r1.b1";
+        observe, place="bpm.26r1.b1";
+        observe, place="bpm.27r1.b1";
+        observe, place="bpm.28r1.b1";
+        observe, place="bpm.29r1.b1";
+        observe, place="bpm.30r1.b1";
+        observe, place="bpm.31r1.b1";
+        observe, place="bpm.32r1.b1";
+        observe, place="bpm.33r1.b1";
+        observe, place="bpm.34r1.b1";
+        observe, place="bpm.33l2.b1";
+        observe, place="bpm.32l2.b1";
+        observe, place="bpm.31l2.b1";
+        observe, place="bpm.30l2.b1";
+        observe, place="bpm.29l2.b1";
+        observe, place="bpm.28l2.b1";
+        observe, place="bpm.27l2.b1";
+        observe, place="bpm.26l2.b1";
+        observe, place="bpm.25l2.b1";
+        observe, place="bpm.24l2.b1";
+        observe, place="bpm.23l2.b1";
+        observe, place="bpm.22l2.b1";
+        observe, place="bpm.21l2.b1";
+        observe, place="bpm.20l2.b1";
+        observe, place="bpm.19l2.b1";
+        observe, place="bpm.18l2.b1";
+        observe, place="bpm.17l2.b1";
+        observe, place="bpm.16l2.b1";
+        observe, place="bpm.15l2.b1";
+        observe, place="bpm.14l2.b1";
+        observe, place="bpm.13l2.b1";
+        observe, place="bpm.12l2.b1";
+        observe, place="bpm.11l2.b1";
+        observe, place="bpm.10l2.b1";
+        observe, place="bpm.9l2.b1";
+        observe, place="bpm.8l2.b1";
+        observe, place="bpm.7l2.b1";
+        observe, place="bpmr.6l2.b1";
+        start, x=0.0002, px=0, y=0.0002, py=0, t=0, pt=0;
+        run, turns=1023;
+    endtrack;
+    ```
+
+## Preparing Data for Analysis
+
+Analysis in `omc3` is mostly done from measurement data, but can also be done on simulated tracking data.
+While `omc3` codes can read data from many machine formats, we will for demonstration purposes first convert our data to the LHC's [SDDS][sdds] binary format.
+
+For this `omc3` provides the `tbt_converter` entrypoint.
+A typical use consists in specifying the location of your turn-by-turn measurement files, the data type of said files and the location in which to write the converted `SDDS` data.
+The formats supported by the converter are:
+
+- Machine formats: `lhc`, `iota` and `esrf`, or any machine using one of these.
+- Simulation formats: the `trackone` and `ptc` formats from tracking data from `MAD-X` or `PTC`.
+
+??? info "The `trackone` and `ptc` formats"
+    What is referred to here as the `trackone` or `ptc` format is obtained by giving the `DUMP` and `ONETABLE` options to the `TRACK` or `PTC_TRACK` commands, respectively, in `MAD-X`.
+    This way, all tracking data is written to a single file on disk, which you can feed to the `tbt_converter`.
+    Remembering which tracking module was used is important as `PTC_TRACK` does not write in the same format as `MAD-X`'s `TRACK`.
```

Comment:
I'd have to check in more detail, but from what I remember, MAD-X and PTC `trackone` are similar enough that either code should work (see also doc of `reader_trackone`). The difference is mainly in the processing, I believe only `reader_trackone` can handle arbitrary long files, whereas `reader_ptc` runs into RAM problems at some point.